service/controller/resource/tccp/spec.go

```diff
@@ -1,6 +1,99 @@
 package tccp
 
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+)
+
+const (
+	// asgMaxBatchSizeRatio is the % of instances to be updated during a
+	// rolling update.
+	asgMaxBatchSizeRatio = 0.3
+	// asgMinInstancesRatio is the % of instances to keep in service during a
+	// rolling update.
+	asgMinInstancesRatio = 0.7
+	// defaultEBSVolumeMountPoint is the path for mounting the EBS volume.
+	defaultEBSVolumeMountPoint = "/dev/xvdh"
+	// defaultEBSVolumeSize is expressed in GB.
+	defaultEBSVolumeSize = "100"
+	// defaultEBSVolumeType is the EBS volume type.
+	defaultEBSVolumeType = "gp2"
+	// rollingUpdatePauseTime is how long to pause ASG operations after creating
+	// new instances. This allows time for new nodes to join the cluster.
+	rollingUpdatePauseTime = "PT15M"
+	// logEBSVolumeMountPoint is the path for mounting the log EBS volume
```

Comment:
Half of these have `.` at the end half of them don't. Consistency for my precious eyes please.